client/Windows/wf_event.c

```diff
@@ -130,15 +130,17 @@ LRESULT CALLBACK wf_ll_kbd_proc(int nCode, WPARAM wParam, LPARAM lParam)
 				DEBUG_KBD("keydown %d scanCode 0x%08lX flags 0x%08lX vkCode 0x%08lX",
 				          (wParam == WM_KEYDOWN), p->scanCode, p->flags, p->vkCode);
 
-				if (wfc->fullscreen_toggle &&
-				    ((p->vkCode == VK_RETURN) || (p->vkCode == VK_CANCEL)) &&
-				    (GetAsyncKeyState(VK_CONTROL) & 0x8000) &&
-				    (GetAsyncKeyState(VK_MENU) & 0x8000)) /* could also use flags & LLKHF_ALTDOWN */
+
+				//if (wfc->fullscreen_toggle && p->vkCode == VK_SCROLL)
+				if (wfc->fullscreen_toggle && p->vkCode == VK_RETURN)
```

Comment:
Why remove the check for `VK_CANCEL`?

Reply:
Adding VK_CANCEL as an 'or' operator will also work.

'''
if (wfc->fullscreen_toggle && (p->vkCode == VK_RETURN || p->vkCode == VK_CANCEL))
'''